lfx-mentorship/2023/01-Mar-May/README.md

```diff
@@ -460,6 +483,29 @@ Note: This item is a work in progress. The selected mentee is expected to contin
 - Upstream Issue: https://github.com/kubeedge/ianvs/issues/48
 - LFX URL: https://mentorship.lfx.linuxfoundation.org/project/50cdbd65-e0cd-4c0f-8c63-6bd5c603ba89
 
+#### Meshery
+
+##### Distributed workflow engine
+
+- Description: Integrate a new architectural component into Meshery: a workflow engine. This project involves shifting Meshery off of bitcask and off of sqlite over to postgres using gorm (golang). Interns will familiarize with concepts of orchestration engines, including chaining workflows, and content lifecycle management.
+- Recommended Skills: Golang, Temporal, ReactJS
+- Mentor(s): Lee Calcote (@leecalcote, leecalcote@gmail.com), Ashish Tiwari (ashishjaitiwari15112000@gmail.com)
+- Issue: <https://github.com/meshery/meshery/issues/3934>
+  
+##### Multi-user cloud native playground
+
+- Description: Advance the cloud native playground in which any CNCF project can be explored. Meshery’s genesis is that of helping teach people about cloud native technology and enabling to operate various types of cloud native infrastructure confidently. The proposed project is aimed at furthering this mission by infusing multi-user collaboration as a pervasisve feature so that users can learn together in a running instance of Meshery.
+- Recommended Skills: ReactJS, CSS, Golang (nice-to-have)
+- Mentor(s): Lee Calcote (@leecalcote, leecalcote@gmail.com), Abhishek Kumar (@abhishek-kumar09, abhimait1909@gmail.com)
+- Issue: <https://github.com/layer5io/meshery/issues/2931>
```

Comment:
We can't accept this project proposal as is -- the GitHub issue is linking to a private Layer 5 repo.

Reply:
Oh, this is an old reference to where the Meshery project was originally located, but post-donation was moved from `layer5io/meshery` to  `meshery/meshery`. Funky, that the GitHub redirection isn't working. 

I went ahead and created a fresh issue: https://github.com/meshery/meshery/issues/7020